person_module.rb

```diff
@@ -0,0 +1,10 @@
+module Person
+# IN HERE WOULD BE TRAITS THAT ALL THREE CLASSES HAVE IN COMMON, SUCH AS NAME, AGE, OFFER_HIGH_FIVE METHOD AND MAYBE SET PHASE METHOD.
+  @age = options.fetch(:age, 0)
+  @name = options.fetch(:name, "")
```

Comment:
You can't set instance variables outside of a method.  Normally these would be in an initialize method.  Modules don't have initialize methods.  That might be a clue that this should not be a module.